contrib/hook.sh

```diff
@@ -53,7 +54,10 @@ main() {
   cp $full_index_path $minimal_index_path $WWW_PATH/
 
   # Update tar.xz archive
-  tar cJf $archive_path _map ??/*.json
+  echo "_map" > $tar_tempfile
+  find . -path "./??/*.json" -print >> $tar_tempfile
+  tar cJf "$archive_path" --files-from=$tar_tempfile
```

Comment:
It should be possible to pass the list of files to `tar` via `STDIN` instead of using a temporary file with `(echo _map && find . -path "./??/*.json" -print) | tar cJf "$archive_path" --files-from=-`

Reply:
Yes, but we reached the shell's default argument length limit, which is what prompted the change. See the snippet from above:
```
...
/app/hook.sh: line 56: /bin/tar: Argument list too long
```

Reply:
My suggestion similarly avoids passing the list of files as arguments by using `--files-from`, but passes `find`'s output to `tar`'s `STDIN` using pipes instead of a temporary intermediate file.

Reply:
Ah, sorry, I see what you meant. Tested and works nicely.